src/utils/translations.js

```diff
@@ -0,0 +1,13 @@
+
+export function areNavButtonsLabelsInConfig(config) {
```

Comment:
This function is too specific. Indeed, you got the gist of the problem that we are facing when we start using keys from a translation config object: to merge the values with default config. Besides the fact that there is still a misunderstandig of the item/manifest terms usage across the whole PR, we can keep this way of identifying which translation to use for now. It just should not rely on counting of keys and so on since it's all dynamic. We cannot know how many keys project configs will provide. What we know for sure is the name of the key, this is the info that we should make use of.